src/backend/gporca/libgpopt/src/xforms/CXformFactory.cpp

```diff
@@ -371,7 +370,7 @@ CXformFactory::Init()
 	GPOS_CATCH_EX(ex)
 	{
 		// destroy memory pool if global instance was not created
-		CMemoryPoolManager::GetMemoryPoolMgr()->Destroy(mp);
+		CMemoryPoolManager::Destroy(mp);
 		m_pxff = NULL;
```

Comment:
Do we really need to set the static variable to NULL (and at other classes uses the same approach, does it really needed, related to the patch?)? In case of raised exception the variable, the value of the static variable, seems  doesn't change, firstly it inited globally with NULL and after the Shutdown method implicitly set it to NULL.

Reply:
This command seems useless, but let it be